modules/k8s/tanka/new-app.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+APP_NAME=$1
+
+NAMESPACE=${NAMESPACE:-""}
+OWNER=${OWNER:-""}
+IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-""}
+IMAGE_TAG=${IMAGE_TAG:-""}
+
+echo ""
+
+if [ -z ${APP_NAME} ]; then
+  echo "Must specify an application name"
+  exit 1
+fi
+if [ -d "lib/${APP_NAME}" ] || [ -d "environments/${APP_NAME}" ]; then
+  echo "Application already exists by the name '${APP_NAME}'"
+  exit 1
+fi
+
+if [ "${NAMESPACE}" = "" ]; then
+  namespaces=$(tk env list --json | jq -r '.[].spec.namespace' | sort | uniq)
+  if [ ${#namespaces[@]} == 1 ]; then
+    NAMESPACE="${namespaces[0]}"
+  else
+    echo "Namespace:"
+    read ns
+    NAMESPACE=$ns
+  fi
+fi
+
+if [ "${OWNER}" = "" ]; then
+  echo "Owner:"
+  read owner
+  OWNER=$owner
+fi
+
+if [ "${IMAGE_REPOSITORY}" = "" ]; then
+  echo "Repository of image to deploy, eg: mintel/portal/portal (note: if not a mintel gitlab repo then you will need to update the 'registry' value manually"
+  read repository
+  IMAGE_REPOSITORY=$repository
+fi
+
+if [ "${IMAGE_TAG}" = "" ]; then
+  echo "Tag of the image to deploy:"
+  read tag
+  IMAGE_TAG=$tag
+fi
+
+mkdir -p lib/${APP_NAME}
+mkdir -p environments/${APP_NAME}/{local,aws.dev,aws.qa,aws.prod}
+
+
+echo """(import 'gitlab.com/mintel/satoshi/kubernetes/jsonnet/sre/libs-jsonnet/mintel-util/main.libsonnet')
+{
+  _config+:: {
+    namespace: '${NAMESPACE}',
+    partOf: '${APP_NAME}',
+    owner: '${OWNER}',
+    name: '${APP_NAME}',
+  },
+
+  appValues:: {
+    global: $._config,
+    image: {
+      repository: '${IMAGE_REPOSITORY}',
+      tag: '${IMAGE_TAG}',
+    },
+    port: 9898,
```

Comment:
Is 9898 our default port? Previously in web-service it used to be 8080 or 8000 (I forget). 9898 comes from podinfo :)

Reply:
yeah, went back and forth on this

tempted to make it another prompt

Reply:
added prompt :)